src/Engine/UnitActionType.java

```diff
@@ -17,15 +17,16 @@
   public static final UnitActionType CAPTURE = new Capture();
   public static final UnitActionType RESUPPLY = new Resupply();
   public static final UnitActionType WAIT = new Wait();
+  public static final UnitActionType DELETE = new Delete();
   public static final UnitActionType LOAD = new Load();
   public static final UnitActionType JOIN = new Join();
 
-  public static final UnitActionType[] GENERIC_ACTIONS = { ATTACK, UNLOAD, CAPTURE, RESUPPLY, WAIT, LOAD, JOIN };
+  public static final UnitActionType[] GENERIC_ACTIONS = { ATTACK, UNLOAD, CAPTURE, RESUPPLY, WAIT, DELETE, LOAD, JOIN };
```

Comment:
Nooo, conflicts. I still have this deleted in `ActionReworkMod`. Ah well.

Reply:
Too little, too late.